src/libraries/System.Text.Json/tests/System.Text.Json.SourceGeneration.UnitTests/JsonSourceGeneratorTests.cs

```diff
@@ -165,15 +171,19 @@ public void TypeDiscoveryWithRenamedAttribute()
             using System.Text.Json.Serialization;
             using ReferencedAssembly;
 
-            using @JsonSerializable = System.Runtime.Serialization.ContractNamespaceAttribute;
+            using @JsonSerializable = System.Runtime.Serialization.CollectionDataContractAttribute ;
```

Comment:
Why use S.R.S.CollectionDataContractAttribute here?

Reply:
I changed to this because, like `JsonSerializableAttribute`,  it is applied to `class`es.

Reply:
This test is modeling type aliasing. In this case, we have two attributes switching names. The test makes sure that our check for `[JsonSerializable(Type)]` can guard against such tricks.

Reply:
I see. A comment here about that would be useful. Thanks